src/pyk/kcfg/kcfg.py

```diff
@@ -422,22 +458,37 @@ def _print_subgraph(indent: str, curr_node: KCFG.Node, prior_on_trace: List[KCFG
                 return
             successor = successors[0]
 
-            if type(successor) is KCFG.Split:
+            if isinstance(successor, KCFG.MultiEdge):
                 ret_lines.append(('unknown', [f'{indent}┃']))
-
-                for target, csubst in successor.targets[:-1]:
-                    ret_edge_lines = _print_split_edge(csubst)
-                    ret_edge_lines = [indent + '┣━━┓ ' + ret_edge_lines[0]] + add_indent(
-                        indent + '┃  ┃ ', ret_edge_lines[1:]
-                    )
+                if type(successor) is KCFG.NDBranch:
+                    ret_lines.append(('unknown', [f'{indent}┃ (1 step)']))
```

Comment:
```suggestion
                multiedge_label = '1 step' if type(successor) is KCFG.NDBranch else 'branch'
                ret_lines.append(('unknown', [f'{indent}┃ ({multiedge_label})']))
```

Maybe this would look nice?